pages/change-password-forgot/index.vue

```diff
@@ -22,6 +22,7 @@
       />
       <ResetPasswordSuccess
         v-else-if="display === 'success'"
```

Comment:
minor : Hello @yoslie , I think better use `v-else` than `v-else-if`. What do you think?

Reply:
sure it is the same, i use this because two reasons:
- i want it to be specific, so only `success` condition will display it
- second, i will use v-else in case there is an error display